### ISSUE TITLE
find way to align the cards to the end of the row

### ISSUE THREAD

**Issue by DinisCruz** [photobox/pbx-group-security#144]
in 

![image](https://user-images.githubusercontent.com/656739/34502549-d6ab42d8-f00b-11e7-9f9d-2ef3656538f2.png)

**Comment by fzipi** [comment]
@DinisCruz Vertical align: top, middle, bottom?

**Comment by DinisCruz** [comment]
the idea is that the boxes should all have the same size

Same prob here

![image](https://user-images.githubusercontent.com/656739/34590399-608e79d6-f1ae-11e7-974e-750f36d65ae1.png)

**Comment by DinisCruz** [comment]
I think the solution is something related to flex https://yoksel.github.io/flex-cheatsheet/

**Comment by RandomFractals** [comment]
just to confirm: make all the cards same height? and update all pages where this happens?

**Comment by RandomFractals** [comment]
@DinisCruz more up to date CSS flexbox guides just released on MDN to review for fixing this:

https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout

I can prototype a fix next week.

Side note: in general, I don't like the idea of mixing bootstrap grid layout with flex. It might require some thorough testing across all browsers and platforms, but the current site is heavily dependent on this already, so we'll have to do both :(

**Comment by DinisCruz** [comment]
yes the idea is see what it looks like when the cards are all the same height

And yes lets try to keep this as bootstrap as possible

btw, I remember fixing this issue already once (for the participants cards you can see at https://owaspsummit.org/website/participants.html)

**Comment by RandomFractals** [comment]
also, this other issue from #241 is related:

reported by @DinisCruz: The cards used in the table view needs to have a min view since they do't wrap up well on small or mobile views (see below)

![pbx-challengeslistcardsissue](https://user-images.githubusercontent.com/656833/34926067-c13ca796-f972-11e7-8778-2c70eb3d4862.png)

**Comment by RandomFractals** [comment]
In general, all of these 'list card' pages need to be reworked for height and row wrapping on mobile.

We have hardcoded 3 cards per row for them and as reported that will not scale on smaller screens :)

**Comment by RandomFractals** [comment]
similar page from the other site by Dinis:

https://github.com/OWASP/owasp-summits/blob/master/website/_includes/blocks/participants.html

**Comment by regisphilibert** [comment]
Hi there, I'm new here ;)
Is it ok if IE11 users still see the boxes as is. If yes, you guys should use css grid to style this grid.
Works like a charm, very few lines fo codes. I'm new to the project and could be wrong, but if you allow I could investigate.

**Comment by regisphilibert** [comment]
https://github.com/project-cx/pbx-group-security/pull/265

**Comment by RandomFractals** [comment]
@regisphilibert flexbox cards height and wrapping split looks good, but I think we'll need another patch for this to have at least 2 or 3 cards for a 900px wide screens in portrait mode. Here is what I got reviewing it on site. We could fit more cards there. Let's iterate on this on Friday to finalize this for all such slim list cards pages. Looks good on wider and mobile screens.

![pbx-challenges-cards-list-portrait](https://user-images.githubusercontent.com/656833/35067729-7175c722-fb9a-11e7-9b0d-dc3acde96b0e.png)

**Comment by regisphilibert** [comment]
Sure lots of possibilities. Couldn't find any breakpoint directions.
Is there any design/layout we can refer too? 
Any person in charge of advising us on integration direction?

**Comment by RandomFractals** [comment]
we are still working on design to dev workflow, but if it looks a bit off to you, we should take a proactive action and fix it. I think we can fit more slim list cards on that screen in portrait desktop view mode. 

Let's revise it in the next rev.
